x/token/keeper/keeper.go

```diff
@@ -50,18 +59,24 @@ func (k Keeper) GetBalance(ctx sdk.Context, owner sdk.AccAddress) sdk.Int {
 		k.cdc.MustUnmarshalBinaryBare(balance, &amount)
 		return amount
 	}
+
+	// check account exists
+	if k.accountKeeper.GetAccount(ctx, owner) != nil {
```

Comment:
@mikluke not 100% sure that we can do this in multinode env

Reply:
Getter which is setter?
I believe it'll force us to consensus errors. I think this code should be in ante

Reply:
cosmos creates new accounts the same way

Reply:
moved init balance to the SetTokenBalance method